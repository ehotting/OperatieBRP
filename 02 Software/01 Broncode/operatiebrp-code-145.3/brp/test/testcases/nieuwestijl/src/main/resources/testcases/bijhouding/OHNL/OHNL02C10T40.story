Meta:
@regels                 R2031
@status                 Klaar
@usecase                UCS-BY.HG

Narrative:
R2031 Bij einde relatie met Land/gebied ongelijk aan Nederland zijn geen Nederlandse locatiegegevens toegestaan

Scenario:   Gemeente einde en Woonplaatsnaam einde niet ingevuld bij het einde van een relatie in het buitenland
            LT: OHNL02C10T40

Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL/OHNL-Anne.xls

Given pas laatste relatie van soort 1 aan tussen persoon 590984809 en persoon 773201993 met relatie id 2000099 en betrokkenheid id 2000100

When voer een bijhouding uit OHNL02C10T40.xml namens partij 'Gemeente BRP 1'

Then is het foutief antwoordbericht gelijk aan /testcases/bijhouding/OHNL/expected/OHNL02C10T40.txt

Then is in de database de persoon met bsn 590984809 wel als PARTNER betrokken bij een HUWELIJK