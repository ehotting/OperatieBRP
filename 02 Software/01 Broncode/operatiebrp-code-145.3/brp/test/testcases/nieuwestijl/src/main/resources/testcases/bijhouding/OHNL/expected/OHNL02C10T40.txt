cvc-complex-type.2.4.a: Invalid content was found starting with element 'brp:landGebiedEindeCode'. One of '{"http://www.bzk.nl/brp/brp0200":gemeenteEindeCode}' is expected.